tests/Kernel/Paragraphs/ParagraphsTest.php

```diff
@@ -260,63 +235,76 @@ public function testListItem(): void {
-    $this->assertCount(3, $meta_elements);
-    $this->assertEquals('Meta 1', trim($meta_elements->getNode(0)->nodeValue));
-    $this->assertEquals('Meta 2', trim($meta_elements->getNode(1)->nodeValue));
-    $this->assertEquals('Meta 3', trim($meta_elements->getNode(2)->nodeValue));
+    $this->assertEquals('Druplicon', $image_element->attr('aria-label'));
 
-    // The primary image markup should not be rendered.
-    $this->assertCount(0, $crawler->filter('.ecl-list-item__primary img.ecl-image'));
-    // The date field should not be rendered.
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__week-day'));
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__day'));
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__month'));
+    $this->assertEquals('Meta 1 | Meta 2 | Meta 3', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__meta')->text()));
 
-    // Change the variant to date.
-    $paragraph->get('oe_paragraphs_variant')->setValue('date');
+    // Change the variant to thumbnail secondary.
+    $paragraph->get('oe_paragraphs_variant')->setValue('navigation');
     $paragraph->save();
 
     $html = $this->renderParagraph($paragraph);
     $crawler = new Crawler($html);
 
-    $this->assertCount(1, $crawler->filter('.ecl-list-item.ecl-list-item--date'));
-    $this->assertEquals('Item title', trim($crawler->filter('.ecl-list-item__title')->text()));
-    $this->assertEquals('Item description', trim($crawler->filter('.ecl-list-item__detail')->text()));
+    $this->assertEquals('Item title', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__title')->text()));
+    $this->assertEquals('Item description', trim($crawler->filter('article.ecl-content-item div.ecl-content-item__description')->text()));
 
-    $link_element = $crawler->filter('.ecl-list-item__link');
+    $link_element = $crawler->filter('article.ecl-content-item div.ecl-content-item__title a.ecl-link');
     $this->assertCount(1, $link_element);
     $this->assertEquals('http://www.example.com/', $link_element->attr('href'));
 
-    $this->assertEquals('Thu', trim($crawler->filter('.ecl-date-block__week-day')->text()));
-    $this->assertEquals('24', trim($crawler->filter('.ecl-date-block__day')->text()));
-    $this->assertEquals('Sep', trim($crawler->filter('.ecl-date-block__month')->text()));
-    $this->assertEquals('1981', trim($crawler->filter('.ecl-date-block__year')->text()));
+    $this->assertCount(1, $crawler->filter('article.ecl-content-item > div'));
+    $this->assertCount(0, $crawler->filter('article.ecl-content-item > div.ecl-u-d-lg-block'));
+    $this->assertCount(0, $crawler->filter('article.ecl-content-item div.ecl-content-item__meta'));
 
-    // No images should be rendered in this variant.
-    $this->assertCount(0, $crawler->filter('img.ecl-image'));
-    // Neither the metas.
-    $this->assertCount(0, $crawler->filter('.ecl-meta__item'));
```

Comment:
We have to keep these assertions.

Reply:
Already on the code, check on the top. I will add code comments to be more clear.